deployment/import_instructions.md

```diff
@@ -26,25 +33,27 @@ cognito.UserPool -
 1. Create just the new cdk stack without anything inside of it.
 
 ```bash
-WITHOUT_IMPORTS=true ./run deploy --stage <YOUR_BRANCH_NAME>
+WITHOUT_IMPORTS=true FORM_ANSWERS=FILL_IN_WITH_VALUES_ABOVE FORM_QUESTIONS=FILL_IN_WITH_VALUES_ABOVE FORM_TEMPLATES=FILL_IN_WITH_VALUES_ABOVE FORMS=FILL_IN_WITH_VALUES_ABOVE STATE_FORMS=FILL_IN_WITH_VALUES_ABOVE STATES=FILL_IN_WITH_VALUES_ABOVE AUTH_USER=FILL_IN_WITH_VALUES_ABOVE ./run deploy --stage <YOUR_BRANCH_NAME>
```

Comment:
The reason this is working this way is that if we import, we have to keep the old names which is not great because you always have to remember the old names and wouldn't match the pattern for new stacks we make with the same cdk code. So I'm taking the dynamo tables we would normally import and using point-in-time-recovery to create an identical table correctly named and managed by the cdk code. I've tried to do it such that in the future we can delete the import code someday and there will be no indication that it ever existed and there will be no weirdness from unconventionally named tables.